### DISCUSSION TITLE
How to set the slidesPerView with partial slides on both ends using SwiperJS?

### DISCUSSION THREAD

**Discussion by graydirt** [nolimits4web/swiper#7404] [Q&A]
Im new to SwiperJS. Right now my code is displaying 3 visible items with partial items on both sides. However I want 4 visible items on it instead of 3.

<img width="676" alt="image" src="https://github.com/nolimits4web/swiper/assets/1822445/0bce2310-4ee6-4e07-82c8-d738668c9867">

I don't know if theres something that I missed on `swiper-api `like `slidesPerView`, `slidesPerGroup` or anything else.

Reproduction link
[https://codepen.io/graydirt/pen/VwNmaOd](https://codepen.io/graydirt/pen/VwNmaOd)

Here is also my JS code for that gallery section.
```javascript
var swiper2 = new Swiper(".mySwiperNav", {
  spaceBetween: 16,
  slidesPerView: 4.5,
  slidesPerGroup: 1,
  centeredSlides: true,
  loop: true,
  /*loopAdditionalSlides: 6,*/
  freeMode: true,
  watchSlidesProgress: true,
  allowTouchMove: true,
  keyboard: {
    enabled: true,
  },
  navigation: {
    nextEl: ".swiper-button-next",
    prevEl: ".swiper-button-prev",
  },
});
var swiper = new Swiper(".mySwiperFor", {
  spaceBetween: 16,
  effect: "fade",
  thumbs: {
    swiper: swiper2,
  },
});
```

**Comment by eswar-janjanam** [comment]
I think with `centeredSlides` set to `true` you cannot do that, because we can center a particular slide only when we have odd number of slides set per view. May be you can try `slidesPerView: 5.5`. In this way, the 3rd slide will be centered with partial slides visible on both sides.
